src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -689,9 +709,28 @@ void AppleseedRendererPBlockAccessor::Set(
         settings.m_material_preview_quality = v.i;
         break;
         
-     //
-     // Image Sampling.
-     //
+      case ParamIdEnableOverrideMaterial:
+        settings.m_enable_override_material = v.i > 0;
+        break;
+
+      case ParamIdOverrideMaterial:
+        if (v.r != nullptr)
+            static_cast<Mtl*>(v.r)->Update(0, FOREVER);
```

Comment:
I had to add this update call here because otherwise when scene was opened with saved override material it didn't render material properly. It looked like material was not updated before the first use.

Reply:
Can you add a comment in the code explaining this?